### DISCUSSION TITLE
Tackling images comping from a json/markdown data source?

### DISCUSSION THREAD

**Discussion by Blakeinstein** [zerodevx/svelte-img#38] [General]
My sveltekit app is wired to pick up image URLs from either a json data set, or Markdown frontmatter. Is there a proposed approach to using svelte-img in such scenarios? Current they are hardcoded to pick up links from the static directory.

**Comment by zerodevx** [comment]
Generally for image transformations to work, two conditions must be met:

1. Image must exist locally (stored in your file system);
2. Image path must be known at **build** time.

For cases that don't meet the above, it probably means that a CDN approach (eg. Cloudinary) is more apt.

Does this help?
